### ISSUE TITLE
Improve link styles settings

### ISSUE THREAD

**Issue by beafialho** [WordPress/gutenberg#44130]
Currently, it's possible to manage the typography settings for different separate elements, including Links and Headings, but if I select a typeface with links such as post terms in mind, this will also adopt that typeface in a paragraph with a different font, which isn't intended. Here's an example:

<img width="525" alt="Captura de ecrã 2022-09-13, às 16 37 52" src="https://user-images.githubusercontent.com/65220155/189945763-08ca1589-9a22-4499-bc66-32297cc1992f.png">

**Comment by mtias** [comment]
I think this is expected, no? If you only want to change post terms you'd add it to that block type. If you modify the link typography globally, all links will try to inherit unless overwritten.

**Comment by tyxla** [comment]
I agree that this is the intended behavior. Will close, but feel free to reopen if something doesn't feel right. Thanks!
